shared/ui-components/package.json

```diff
@@ -27,6 +27,7 @@
     "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-require-context-hook": "^1.0.0",
+    "@emotion/styled": "10.0.23",
```

Comment:
Even though we're not using emotion, Storybook was giving me grief for some reason until I added this. Saw some chatter on GitHub this was something other people were running into in recent SB versions as well. 🤷‍♂